actionpack/lib/action_dispatch/http/response.rb

```diff
@@ -168,10 +163,16 @@ def self.merge_default_headers(original, default)
     # The underlying body, as a streamable object.
     attr_reader :stream
 
-    def initialize(status = 200, header = {}, body = [])
+    def initialize(status = 200, headers = nil, body = [])
       super()
 
-      @header = Header.new(self, header)
+      @headers = Headers.new
+
+      if headers
+        headers.each do |key, value|
```

Comment:
`headers&.each` assuming that is allowed by the style guide.

Reply:
Thanks, I've updated the relevant PR, that is good feedback.